src/WatchDogs.Domain/Domain.csproj

```diff
@@ -6,4 +6,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Again, please remove these Nugets. They are currently not needed.

Reply:
Done.

Reply:
Done.